### ISSUE TITLE
Allow optional treatment of functions as values

### ISSUE THREAD

**Issue by vitaly-t** [vitaly-t/path-value#3]
Consider extending the standard syntax with something to treat functions as values, i.e. do not invoke them.

Example:

```js
const obj = {
    first: Date
};

const now = resolveValue(obj, 'first.now');
```

---

For example, allow `^` in front of a property, so for the case above we can write the following:

```js
const now = resolveValue(obj, '^first.now');
```

This would tell not to invoke `first` when it is a function, but rather stay above it, i.e. treat it as a value.

**Comment by vitaly-t** [comment]
Implemented this in branch `func-value`, though not sure about merging it, because it is a step off from JavaScript native syntax, combined with very small usage cases.

**Comment by vitaly-t** [comment]
Implemented and released with `v0.4.0`.

Now, whenever you want to override the default behavior for a specific property-function, you can prefix it with `^`.

Example from the main page:

```js
// Avoid calling Date() below, treat Date function as a value:

resolveValue(global, '^Date.now'); //=> 1608729428776 (current time tick)
```
